db/migrate/20160505005253_add_send_count_to_invitations.rb

```diff
@@ -0,0 +1,5 @@
+class AddSendCountToInvitations < ActiveRecord::Migration
+  def change
+    add_column :invitations, :send_count, :integer, null: true, default: 0
```

Comment:
Nice catch +1

On Tue, May 10, 2016 at 6:57 AM, Robert Guthrie <notifications@github.com>
wrote:

> In db/migrate/20160505005253_add_send_count_to_invitations.rb
> <https://github.com/loomio/loomio/pull/3342#discussion_r62580380>:
>
> > @@ -0,0 +1,5 @@
> > +class AddSendCountToInvitations < ActiveRecord::Migration
> > +  def change
> > +    add_column :invitations, :send_count, :integer, null: true, default: 0
>
> I think we decided null was going to be false, ae?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/3342/files/52b119677cef00a2725f01baea3586b111eeb738#r62580380>
>